Index: src/client_main/java/GUI_client/CheckersApplication.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package client_main.java.GUI_client;\r\n\r\nimport javafx.application.Application;\r\nimport javafx.fxml.FXMLLoader;\r\nimport javafx.scene.Scene;\r\nimport javafx.stage.Stage;\r\n\r\nimport java.io.IOException;\r\n\r\npublic class CheckersApplication extends Application {\r\n    @Override\r\n    public void start(Stage stage) throws IOException {\r\n        FXMLLoader fxmlLoader = new FXMLLoader(CheckersApplication.class.getResource(\"/GUI_client/checkers.fxml\"));\r\n        Scene scene = new Scene(fxmlLoader.load(), 900, 600);\r\n        stage.setResizable(false);\r\n        stage.setTitle(\"Checkers\");\r\n        stage.setScene(scene);\r\n        stage.show();\r\n    }\r\n\r\n    public static void main(String[] args) {\r\n        launch();\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/client_main/java/GUI_client/CheckersApplication.java b/src/client_main/java/GUI_client/CheckersApplication.java
--- a/src/client_main/java/GUI_client/CheckersApplication.java	(revision c62d96c078902a541ab86fbc2c95572bbaa8955d)
+++ b/src/client_main/java/GUI_client/CheckersApplication.java	(date 1743384739497)
@@ -1,4 +1,4 @@
-package client_main.java.GUI_client;
+package GUI_client;
 
 import javafx.application.Application;
 import javafx.fxml.FXMLLoader;
